raiden/tests/utils/smoketest.py

```diff
@@ -227,3 +229,41 @@ def start_ethereum(smoketest_genesis):
         init_log_err=init_err,
     )
     return ethereum_node, ethereum_config
+
+
+def patch_smoke_fns(fn):
```

Comment:
:(

Reply:
This will go away. I will make an issue as soon as this is merged, to deploy the contracts normally also in the smoke tests. Promise. Just judged this was faster for the moment.